src/sas/qtgui/Perspectives/Corfunc/UI/CorfuncPanel.ui

```diff
@@ -393,6 +393,13 @@
          </property>
         </widget>
        </item>
+       <item>
+        <widget class="QPushButton" name="cmdSave">
+         <property name="text">
+          <string>Save</string>

```

Comment:
Would be useful to be more descriptive - what is being saved? Maybe add a tooltip?

Reply:
I've added tooltips for the main corfunc buttons.